pkg/jx/cmd/create_vault.go

```diff
@@ -325,13 +331,13 @@ func (o *CreateVaultOptions) createVaultBucket(vaultName string) (string, error)
 	return bucketName, nil
 }
 
-func (o *CreateVaultOptions) createVaultAuthServiceAccount(vaultName string) (string, error) {
+func (o *CreateVaultOptions) createVaultAuthServiceAccount(vaultName string, clusterName string) (string, error) {
 	client, _, err := o.KubeClient()
 	if err != nil {
 		return "", errors.Wrap(err, "creating kubernetes client")
 	}
 
-	serviceAccountName := fmt.Sprintf("%s-auth-sa", vaultName)
+	serviceAccountName := gke.VaultAuthServiceAccountName(vaultName, clusterName)
```

Comment:
should this be tied to GKE here? others that use GKE have GKE in the function name so it is obvious